packages/example/pubspec.lock

```diff
@@ -334,84 +334,77 @@ packages:
       path: "../integrations/rudder_integration_adjust_flutter"
       relative: true
     source: path
-    version: "1.0.7"
+    version: "1.1.0"
   rudder_integration_amplitude_flutter:
     dependency: "direct main"
     description:
       path: "../integrations/rudder_integration_amplitude_flutter"
       relative: true
     source: path
-    version: "1.0.7"
+    version: "1.1.0"
   rudder_integration_appcenter_flutter:
     dependency: "direct main"
     description:
       path: "../integrations/rudder_integration_appcenter_flutter"
       relative: true
     source: path
-    version: "1.1.7"
+    version: "1.2.0"
   rudder_integration_appsflyer_flutter:
     dependency: "direct main"
     description:
       path: "../integrations/rudder_integration_appsflyer_flutter"
       relative: true
     source: path
-    version: "1.1.4"
+    version: "1.1.5"
   rudder_integration_braze_flutter:
     dependency: "direct main"
     description:
       path: "../integrations/rudder_integration_braze_flutter"
       relative: true
     source: path
-    version: "1.0.7"
+    version: "1.1.0"
   rudder_integration_firebase_flutter:
     dependency: "direct main"
     description:
       path: "../integrations/rudder_integration_firebase_flutter"
       relative: true
     source: path
-    version: "2.0.6"
-  rudder_integration_leanplum_flutter:
```

Comment:
Bit confused with this oe do we wantto remove it from exampleapp? I there any issue with this destination?

Reply:
This was removed previously due to an issue on the iOS platform. Added it back now.